src/components/Homepage/About/Feature.jsx

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import classes from './Feature.module.css';
+
+export default function Feature({ img, title, descr, reversed }) {
+  return (
+    <div className={reversed ? classes.features__item : `${classes.features__item} ${classes.features__item_reversed}`}>
+      <div className={classes.features__image}>
+        <img src={img} alt='feature-img' />
+      </div>
+      <div className={classes.features__text}>
+        <div className={classes.features__title}>{title}</div>
+        <div className={classes.features__descr}>{descr}</div>
+      </div>
+    </div>
+  );
+}
+
+Feature.propTypes = {
+  img: PropTypes.instanceOf(Object).isRequired,
```

Comment:
Is it object ? I guess it should be a string(url)

Reply:
It is strange, but you're right....

```suggestion
Feature.propTypes = {
  img: PropTypes.string,
  title: PropTypes.string,
  descr: PropTypes.string,
  reversed: PropTypes.bool,
};

Feature.defaultProps = {
  img: '',
  title: '',
  descr: '',
  reversed: false,
};
```